### ISSUE TITLE
Exact Same outputs with 2 different datasets

### ISSUE THREAD

**Issue by aflah02** [tensorflow/tensorflow#55266]
I was trying to reimplement the paper https://arxiv.org/pdf/1901.11196.pdf and created 2 models which have same structure one takes in the augmented data and one doesn't to see which one performs better but for some weird reason both of them have exact same loss and other metrics and it doesn't even change after every epoch
This is the link for my implementation - https://colab.research.google.com/drive/192mGhABi1n51cg8SFLvuUCwvsYIMNvx1?usp=sharing
The model training is at the very end
I tried to check and the inputs, encoders etc. are all different to both models but still this issue persists. Any ideas what could be going wrong?
